scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -544,6 +558,27 @@ object Config {
       .setMapSideAggregationThreshold(100 * 1000) ++ fromHadoop(conf))
       .setSerialization(Right(classOf[serialization.KryoHadoop]))
       .setScaldingVersion
+
+  /**
+   * Add a file to be localized to the config. Intended to be used by user code.
+   * @param uri The uri of the cache to be localized
+   * @param config Config to add the cache to
+   * @return new Config with cached files
+   */
+  def addDistributedCacheFile(uri: URI, config: Config): Config = {
+    val newFile = DistributedCacheFile
+      .symlinkedUriFor(uri)
```

Comment:
there's some logic in DistributedCacheFile for making paths fully qualified -- should we expose those too and and use them here too?